internal/pkg/deploy/cloudformation/stack/task.go

```diff
@@ -0,0 +1,109 @@
+package stack
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/aws/copilot-cli/internal/pkg/deploy"
+	"github.com/aws/copilot-cli/internal/pkg/template"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+)
+
+const (
+	taskTemplatePath = "task/cf.yml"
+
+	TaskNameParamKey         = "TaskName"
+	TaskCPUParamKey          = "TaskCPU"
+	TaskMemoryParamKey       = "TaskMemory"
+	TaskLogRetentionParamKey = "LogRetention"
+
+	TaskContainerImageParamKey = "ContainerImage"
+	TaskTaskRoleParamKey       = "TaskRole"
+	TaskCommandParamKey        = "Command"
+)
+
+type taskStackConfig struct {
+	Name string
+
+	Cpu    int
+	Memory int
+
```

Comment:
We may also need to support EntryPoint, as well.

Reply:
A new flag🚩 Sounds good! Will do this in a separate PR!